Dockerfile

```diff
@@ -71,12 +71,16 @@ RUN npm run build
 # Stage: Production environment
 #
 FROM base AS prod
+ARG image_tag
 ENV NODE_ENV=production
 
 COPY --from=npm-prod /app/ .
 COPY --from=build-prod /app/build/ build/
 
 USER node
 
+LABEL org.opencontainers.image.revision=${image_tag} \
+      org.opencontainers.image.source=https://github.com/libero/article-store
```

Comment:
Been using 4-space indenting rather than alignment.

Reply:
No opinions (but alignment in Terraform is terrible in fact)